zcash_keys/src/keys.rs

```diff
@@ -732,7 +756,7 @@ impl UnifiedFullViewingKey {
     }
 
     /// Returns the string encoding of this `UnifiedFullViewingKey` for the given network.
-    pub fn encode<P: consensus::Parameters>(&self, params: &P) -> String {
```

Comment:
Same thing here, keep `encode` but just have it implemented in terms of `to_ufvk`.